### ISSUE TITLE
Creeperlings AI Broken

### ISSUE THREAD

**Issue by coco9515** [Ladysnake/Creeper-Spores#7]
The AI of the creeperlings broke completly (frozen, invunerable, don't react or move, ect..).

To find and reproduce the bug succesfully :
I had to create creeperlings, die (I did not try but could be just tp or exiting the loaded chunks), come back and see all the creeperlings completly frozen.
My test was close to spawn but should have been outside loaded chunks as I respawn.

It also broke the pillager infected by the spores - in the same manner.
I tried on sheeps, seems fine - need more tests but I don't have the time.

TLDR; after I die, creeperlings AI froze.

**Comment by DietmarKrause** [comment]
perhaps related: https://github.com/AllTheMods/ATM-Fabric/issues/10
in my case the creeperspores dont move either, from that moment on no animal (cows and so on) move either and if i break blocks they disappear,
